client/connection/manager.go

```diff
@@ -43,11 +43,18 @@ func NewManager(mysteriumClient server.Client, dialogEstablisherCreator DialogEs
 	}
 }
 
-func (manager *connectionManager) Connect(consumerID, providerID identity.Identity) error {
+func (manager *connectionManager) Connect(consumerID, providerID identity.Identity) (err error) {
```

Comment:
It seems this method always returns error explicily (i.e. `return err`). Why do we need this var here?

Reply:
Due to the deferred closure, it needs to know whether there was an error